Makefile

```diff
@@ -2,8 +2,9 @@ UNAME_S := $(shell uname -s)
 build: 
 	go build ./cmd/web3
 
-install:
-	go install ./cmd/web3
+install: build
+	# go build -o /usr/local/bin/web3 ./cmd/web3
+	sudo cp web3 /usr/local/bin/web3
```

Comment:
`sudo` seems heavy handed

Reply:
And it's commented out? Leftover hack?

Reply:
sudo is the only way to install it over the main installer. 

Removed comment.

Reply:
I'm not following. Why wasn't `go install` working?

Reply:
Well just tried again, thinking the installed web3 was overriding the one that `go install` makes and throws into ~/go/bin. But looks like I also don't have ~/go/bin in my path so there's that. 😆  But the way it's done here ensures that you won't have conflicting versions installed if you installed it via the install.sh script too.